submissions/balancer-design/universal_8to8.md

```diff
@@ -0,0 +1,88 @@
+## Universal 8-8: Perfectly balanced, as all things should be
+
+Some time ago I made a bunch of computer parts with belts and splitters. This time I will continue my unhealthy obsession with moving items around by making something that's actually at least slightly practical, that being a balancer. Now, why would I write a whole article on designing a balancer? Well that is because the one we will be making today is no average balancer. This article won't go very deep into balancer design principles, since we will be using pre-made and well-known blueprints for the most part. What you *do* need to know is some terminology:
+
+* Input-balanced: Draws from all inputs evenly. Especially important when merging many inputs to a few outputs.
+* Output-balanced: Gives to all outputs evenly. Especially important when splitting a few inputs to many outputs.
+* Throughput-limited: Has internal bottlenecks. An example of a throughput-limited system would be merging two belts into one, and then splitting it back into two. Such a system has a maximum throughput of one belt, even though two belts are available both on input and output.
+* Throughput-unlimited: Opposite of throughput-limited.
+
+All blueprints from this article (and a couple extras) can be found at [this link](https://controlc.com/70909aae).
+
+### The premise
+
+Picture the following: you're a new player, and you've just heard about those things called "balancers" and how useful they are. You see an image of a 4-4 balancer, build it, but then only hook up 3 outputs to get 3 balanced belts. Instead you're met with a 2:1:1 ratio and disappointment:
```

Comment:
You should mention here which balancer the player likely built, and just mention what it should've been to work as they expected.